hogwarts/scss/_article.scss

```diff
@@ -125,9 +125,41 @@ article {
 
     }
   }
-}
+
+  /* ipython notebook specific styles */
+  // Disgusting hack: We are including the minimized ipython css file
+  // which was compiled from less files as scss to make sure it only
+  // applies to the ipython_notebook parts:
+  @import "_ipython_style";
```

Comment:
remove all those lines below + pygments/ipython css and replacing it with

```scss
  // ipython stuff
  .prompt {
      display: none;
  }

  .output_png {
    width: 100%;
    text-align:center
  }

  .output_text {
    pre {
      padding: 0;
    }
  }
```

makes it consistent with the rest of the code samples